src/client/app/components/TooltipHelpComponent.tsx

```diff
@@ -58,6 +58,7 @@ export default function TooltipHelpComponent(props: TooltipHelpProps) {
 		'help.home.compare.sort.tip': { link: `${helpUrl}/compareGraphic.html#usage` },
 		'help.home.error.bar': { link: `${helpUrl}/errorBar.html#usage` },
 		'help.home.export.graph.data': { link: `${helpUrl}/export.html` },
+		'help.home.history': { link: `${helpUrl}/home.html` },
```

Comment:
Could this be changed to history.html since that is where the new help page should go (it does not yet exist). Also, if you want, you can click the box in the PR description that documentation change is then needed.

Reply:
done